internal/photoprism/colors.go

```diff
@@ -1,12 +1,13 @@
 package photoprism
 
 import (
-	"github.com/RobCherry/vibrant"
-	"github.com/lucasb-eyer/go-colorful"
-	"golang.org/x/image/colornames"
 	"image"
 	"os"
 	"sort"
+
```

Comment:
Please make sure to run `make fmt` to avoid changes in the import order (should sort them alphabetically)

Reply:
These import statements have been formatted by the Go standard, `goformat`. Fmt states this about it:

>gofmt follows a simple rule that works for all use cases: imports may be separated by
blank lines, and each set of blank lines is sorted independently.
goimports, which does more than just formatting, applies a different rule: it groups
imports according to whether they are standard imports, appengine imports, or other
imports.  It emits each group separately.  Then the gofmt algorithm kicks in and sorts
each group independently.
The goimports and gofmt programs do different jobs and it is not a goal that they
produce identical results.

Are you saying you don't want `goformat`? :)

Also, instead of a target your IDE should definitely take care of formatting rules as that's a much more convenient thing to have instead of running a make target.

Reply:
You mean `goimports`, not `goformat`? `make fmt` uses `go fmt`.

If `goimports` doesn't cause any issues (does it need installation? will GoLand remove the empty line again?), we can add it to the make target. Just want the imports to be the same for everyone.

Reply:
If everyone uses different IDE settings, merge conflicts and differently formatted code will be the result. That's why we use make. Of course if your IDE can do it the same, use it. Maybe I should also check my IDE settings... not a bad idea.

Reply:
Added it to `make fmt`, see 6a2af16d1cd556d4d25eb2c43c1cd43e9b9dad71 / #50

Reply:
Interesting. I thought goland uses these tools by default. By the way that's why I like Go, formatting is a language standard and thus every tool from vimgo to goland should be applying them.

Yes, you are right. 😊 I meant goimports.

Reply:
No, you need to manually install `goimports` and enable it as a save hook. Should add that to the developer guide. Not a huge fan of extra tools in general: https://twitter.com/lastzero/status/1056992297135603712